subsys/bluetooth/host/gatt.c

```diff
@@ -3192,11 +3192,11 @@ static void sc_restore_rsp(struct bt_conn *conn,
 }
 
 static struct bt_gatt_indicate_params sc_restore_params[CONFIG_BT_MAX_CONN];
+static uint16_t sc_range[2];
```

Comment:
I see the bug this is fixing, but it seems like it's introducing another one: how will you handle doing this simultaneously for multiple connections, if you only have one `sc_range` buffer? I think there is `CONFIG_BT_MAX_CONN` number of `bt_gatt_indicate_params` on the previous line precisely because there needs to be independent contexts for each connection.

Reply:
That is a fair point. 

I guess the fix here just fixes the warning, as the `sc_range` value actually never used in the indicate callback. 

We could make this a 
```c
static uint16_t sc_range[CONFIG_BT_MAX_CONN][2];
```

for more correctness at the cost of some additional memory.

Reply:
Thanks for that. Will fix